src/main/resources/swagger/api-docs.yaml

```diff
@@ -202,6 +202,37 @@ paths:
         500:
           description: Internal Error
   /workspaces/{workspaceNamespace}/{workspaceName}/submissions/{submissionId}:
+    delete:
+      tags:
+      - Submissions
+      operationId: deleteSubmission
+      summary: |
+        delete a submission
+      produces:
+      - application/json
+      parameters:
+        - name: workspaceNamespace
+          description: Workspace Namespace
+          required: true
+          type: string
+          in: path
+        - name: workspaceName
+          description: Workspace Name
+          required: true
+          type: string
+          in: path
+        - name: submissionId
+          description: Submission ID
+          required: true
+          type: string
+          in: path
+      responses:
+        200:
```

Comment:
re "rawls API says this returns 204, not 200. Are you sure?"
I am not sure.  However, I am sure that Greg told me that we are not using RAWLS as a model however.